tslint.json

```diff
@@ -1,41 +1,14 @@
 {
-  "extends": [
-    "tslint:recommended",
-    "react",
-    "tslint-config-prettier"
-  ],
+  "extends": ["tslint:recommended", "react", "tslint-config-prettier"],
   "rules": {
     "arrow-parens": false,
-    "interface-name": [
-      true,
-      "never-prefix"
-    ],
-    "max-classes-per-file": [
-      false
-    ],
-    "member-access": [
-      false,
-      "check-accessor",
-      "check-constructor"
-    ],
-    // Disabled till there’s an auto-fixer for this.
-    // https://github.com/palantir/tslint/blob/master/src/rules/objectLiteralSortKeysRule.ts
+    "interface-name": [true, "never-prefix"],
+    "max-classes-per-file": [false],
+    "member-access": [false, "check-accessor", "check-constructor"],
     "object-literal-sort-keys": false,
+    "ordered-imports": false,
```

Comment:
Was there a discussion about this change anywhere?

Reply:
There was not afaik.

@damassi In our team we agreed a while ago that when we want to make subjective/dev workflow/tooling related changes, we would open a issue for discussion. Can you do so and make a case for the options and your suggestion?

Reply:
Where should I file this?

Reply:
In this case I think here (the emission repo) is perfect and then share it in the CE dev channel for everybody to pitch in.

Reply:
👍